code/index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width , initial-scale=1">
+
+<head>
+  <script src="script.js"></script>
+  <script src="script1.js"></script>
+  <link rel="stylesheet" id="pagestyle" type="text/css" href="style.css">
+  <title>Calculator</title>
+</head>
+
+<body>
+  <h1>Calculator by Roman</h1>
+  <div id="box">
+    <form name="calculator" id="cal">
```

Comment:
Please let's go unify ... names for name and id etc. 
name will always start name ... so ... name-calculator
id will always start id ... so ... id-calculator

Reply:
unified... id with - such as id-calculator same for class just camelCase used for name because of js